dist.ini

```diff
@@ -0,0 +1,72 @@
+name    = perl-Test-Pod-CoverageChange
+author  = Binary Group Services Ltd <BINARY@cpan.org>
+license = Perl_5
+copyright_holder = Binary Group Services Ltd
+copyright_year   = 2020
+main_module= lib/Test/Pod/CoverageChange.pm
+
+[Git::GatherDir]
```

Comment:
Maybe https://metacpan.org/pod/Dist::Zilla::PluginBundle::Author::DERIV already provides for the rest of this?

Reply:
you mean I replace all the content below this line with `[Author::DERIV]`? If the answer is yes I change it a few minutes ago but I'm not able to install the CPAN module for this plugin. Can you please help me with that?

Reply:
Required plugin Dist::Zilla::Plugin::Author::DERIV isn't installed.

Run 'dzil authordeps' to see a list of all required plugins.
You can pipe the list to your CPAN client to install or update them:

    dzil authordeps --missing | cpanm

I noticed that there is a difference between the required module and the module that dzil has tried to install:
// dzil is trying to install this plugin after I use [Author::DERIV]
Dist::Zilla::Plugin::Author::DERIV

// we can install our plugin using this namespace
Dist::Zilla::PluginBundle::Author::DERIV

Reply:
I think its supposed to be

```
[@Author::DERIV]
```

precisely because it is a PluginBundle, not just a single Plugin.

See also https://metacpan.org/pod/Dist::Zilla::Starter